client/src/components/framework/toasters.ts

```diff
@@ -1,40 +1,43 @@
-import { Position, Toaster, Intent } from "@blueprintjs/core";
+import { Position, Toaster, Intent, IToaster } from "@blueprintjs/core";
 
 /* styles */
 // @ts-expect-error ts-migrate(2307) FIXME: Cannot find module './menubar.css' or its correspo... Remove this comment to see the full error message
 import styles from "./toasters.css";
 
 /** Singleton toaster instance. Create separate instances for different options. */
+let ToastTopCenter: IToaster | null = null;
 
-const ToastTopCenter = Toaster.create({
-  className: "recipe-toaster",
-  position: Position.TOP,
-  maxToasts: 4,
-});
+if (typeof document !== "undefined") {
+  ToastTopCenter = Toaster.create({
+    className: "recipe-toaster",
+    position: Position.TOP,
+    maxToasts: 4,
+  });
+}
```

Comment:
in React 18 + Blueprint 5, we'll need to update how this is instantiated here. But we will have example in DP to follow!